### PR TITLE
[new] Add preact-compat w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by fokusferit**
Adding preact-compat@3.15.0 to cdnjs

Pull request for issue: #
Related issue(s): developit/preact-compat#281

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/developit/preact-compat 
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/preact-compat
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 10
   - Count of stars: 320
   - Count of forks: 57
 * NPM download stats (optional):
   - Downloads in the last day: 948
   - Downloads in the last week: 5,524
   - Downloads in the last month: 22,975

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/preact-compat/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "react",
+    "compatibility"
+  ],
+  "homepage": "https://github.com/developit/preact-compat",
+  "npmName": "preact-compat",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(*.+(flow|ts))"
```
Comment (pvnr0082t) [inline ajax/libs/preact-compat/package.json]: @fokusferit We can use `"**/*"` to fetch all files under dist directory.

File: ajax/libs/preact-compat/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "preact-compat",
+  "version": "3.15.0",
```
Comment (pvnr0082t) [inline ajax/libs/preact-compat/package.json]: We don't need version field if adding lib via single package.json. Please remove it. Thanks.

File: ajax/libs/preact-compat/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/developit/preact-compat.git"
+  },
+  "keywords": [
+    "preact",
+    "react",
+    "compatibility"
+  ],
+  "homepage": "https://github.com/developit/preact-compat",
```
Comment (pvnr0082t) [inline ajax/libs/preact-compat/package.json]: We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo.

**Comment by PeterDaveHello** [comment]
@pvnr0082t we don't need to force add author info, it's fine.

**Comment by fokusferit** [comment]
@PeterDaveHello no problem, just added it. Thanks for the review @pvnr0082t :)

**Comment by PeterDaveHello** [comment]
Commits need to be squashed here, give me 2 mins.

**Pull request merged**